src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/iOS/iOSTokenCacheAccessor.cs

```diff
@@ -315,19 +259,6 @@ private SecRecord CreateRecord(string account, string service, string generic, i
             };
         }
 
-        private SecStatusCode Remove(string account, string service, int type)
```

Comment:
>SecStatusCode [](start = 16, length = 13)

i thought we needed this for keychain access

Reply:
None of these methods are called from anywhere within the ADAL codebase.

Reply:
oh...i was in msal...i see. nevermind then.

---
In reply to: [255658016](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1498#discussion_r255658016) [](ancestors = 255658016)